### ISSUE TITLE
Make current navigation bar look better

### ISSUE THREAD

**Issue by Nodrokov** [SeleneSat/SeleneSat.github.io#2]
right now it's just a gray rectangle.  While it works fine, and is certainly better than a links in the body, it could use a little spiffing up.

**Comment by chasewilkinson** [comment]
I like my rounded corners, but they still look plain.

**Comment by Nodrokov** [comment]
Yeah, it definitely looks much better, but I think it's still far from perfect

**Comment by Nodrokov** [comment]
Thanks
